src/test/hammerdb/README.md

```diff
@@ -0,0 +1,15 @@
+
+# How to trigger hammerdb benchmark jobs
+
+You can trigger two types of hammerdb benchmark jobs:
+-ch_benchmark (analytical and transactional queries)
+-tpcc_bnehcmark (only transactional queries)
+
+If need to prepend `ch_benchmark/` or `tpcc_benchmark/` prefix to your branch to trigger these jobs
+relatively.
+
+You will see the results in [https://github.com/citusdata/release-test-results](https://github.com/citusdata/release-test-results).
+
+**You will need to delete the resource groups manually from portal, the resource groups are `citusbot_ch_benchmark_rg` and `citusbot_tpcc_benchmark_rg`.**
```

Comment:
Is this still valid after https://github.com/citusdata/test-automation/pull/168?

Reply:
No updating